src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs

```diff
@@ -1,60 +1,59 @@
-﻿using GroupDocs.Annotation.Domain;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
     public class ResourceRedactionAnnotator : BaseAnnotator
     {
-        public ResourceRedactionAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
-        {
-        }
-        
-        public override AnnotationInfo AnnotateWord()
+        private ResourcesRedactionAnnotation resourcesRedactionAnnotation;
+
+        public ResourceRedactionAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
-            AnnotationInfo resourceRedactionAnnotation = InitAnnotationInfo();
-            return resourceRedactionAnnotation;
+            this.resourcesRedactionAnnotation = new ResourcesRedactionAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)